### PR TITLE
Move useful utils from @pigi/core into @pigi/utils

### LINKED ISSUE #21: Move useful utils into `@pigi/utils`
We have several utilities in [utils.ts](https://github.com/plasma-group/pigi/blob/master/packages/core/src/utils/misc.ts). These seem sufficiently useful that they could probably be moved into [@pigi/utils](https://github.com/plasma-group/pigi/tree/master/packages/utils). 

So this is relatively simple! Move these utils into the [misc](https://github.com/plasma-group/pigi/blob/master/packages/utils/src/utils/misc.ts) utils of `@pigi/utils` and open a PR there, then remove those utils in a PR here :-)

### PR DESCRIPTION
**Pull request by kfichter**
- [x] I have read and understood the [Plasma Group Contributing Guide and Code of Conduct](https://github.com/plasma-group/pigi/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

------------------------------------------------------------------------

Fixes #21 

## Description

Pretty simple. I just took some of the useful stuff from @pigi/core and moved it into @pigi/utils.

### CODE REVIEW

File: packages/utils/src/utils/misc.ts [hunk 1]
```diff
@@ -77,3 +77,56 @@ export const bnMin = (a: BigNum, b: BigNum) => {
...
+ * JSON-stringifies a value if it's not already a string.
+ * @param value Value to stringify.
+ * @returns the stringified value.
+ */
+export const stringify = (value: any): string => {
+  if (!(typeof value === 'string')) {
+    value = JSON.stringify(value)
+  }
+  return value as string
+}
```
[the remark this answers is not in the document]
Comment (karlfloersch) [reply packages/utils/src/utils/misc.ts]: Actually, should this take in multiple inputs? For instance a Buffer & also convert that to a string?

[the remark this answers is not in the document]
Comment (kfichter) [reply packages/utils/src/utils/misc.ts]: Hmm what about `jsonify`?

[the remark this answers is not in the document]
Comment (kfichter) [reply packages/utils/src/utils/misc.ts]: Yeah I like `jsonStringify`

[the remark this answers is not in the document]
Comment (kfichter) [reply packages/utils/src/utils/misc.ts]: > Actually, should this take in multiple inputs? For instance a Buffer & also convert that to a string?

Hmmmm maybe. We could potentially make this change in another issue if we find this to be sufficiently useful.

**Pull request closed without merging**